src/index.html

```diff
@@ -1,15 +1,145 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post" onsubmit="onSubmit()">
+      <fieldset class="block">
+        <legend>Personal information
+        </legend>
+        <div class="input">
+          <label for="surname">Surname:
+            <input type="text"
+            name="surname" id="surname">
```

Comment:
якщо в тега є три і більше атрибутів то кожен атрибут потрібно писати з нової лінії
переглянь весь код і повиправляй відступи для атрибутів
```suggestion
            <input
              type="text"
              name="surname"
              id="surname"
            >
```